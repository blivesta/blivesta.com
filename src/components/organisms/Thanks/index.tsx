import * as React from 'react';

import Title from '../../atoms/Title';
import P from '../../atoms/P';
import ContentBlock from '../../molecules/ContentBlock';

const Thanks = () => {
  return (
    <ContentBlock id="thanks" title="Send complete">
      <Title>Thank you for contact</Title>
      <P lang="en">The message has been sent successfully. We will contact you as soon as possible. Thank you.</P>
      <P lang="ja">送信が完了しました。できるだけ早くご連絡いたします。ありがとうございました。</P>
    </ContentBlock>
  );
};

export default Thanks;
