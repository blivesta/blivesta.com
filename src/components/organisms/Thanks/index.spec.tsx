import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Thanks from '.';

describe('Thanks', () => {
  it('snapshot', () => {
    const tree = shallow(<Thanks />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
