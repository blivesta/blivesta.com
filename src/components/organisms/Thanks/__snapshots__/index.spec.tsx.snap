// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thanks snapshot 1`] = `
<ContentBlock
  id="thanks"
  title="Send complete"
>
  <styled.h1>
    Thank you for contact
  </styled.h1>
  <P
    lang="en"
  >
    The message has been sent successfully. We will contact you as soon as possible. Thank you.
  </P>
  <P
    lang="ja"
  >
    送信が完了しました。できるだけ早くご連絡いたします。ありがとうございました。
  </P>
</ContentBlock>
`;
